node/lib/cmd/commit-shadow.js

```diff
@@ -75,10 +75,16 @@ exports.configureParser = function (parser) {
         required: true,
         help: "commit message for shadow commits",
     });
+    parser.addArgument(["-t", "--timestamp"], {
+        type: Number,
+        required: false,
+        defaultValue: null,
+        help: "timestamp for shadow commits",
```

Comment:
In what format?

Reply:
Do we care about this format? if the user uses the same format, it should generate the same sha.